xml/System/Span`1.xml

```diff
@@ -155,8 +155,8 @@ If `array` is `null`, this constructor returns a `null` `Span<T>`.
       </Parameters>
       <Docs>
         <param name="pointer">A pointer to the starting address of a specified number of bytes in memory.</param>
-        <param name="length">The number of bytes to be included in the <see cref="T:System.Span`1" />.</param>
-        <summary>Creates a new <see cref="T:System.Span`1" /> object  from a specified number of bytes starting at a specified memory address.</summary>
+        <param name="length">The number of <typeparamref name="T"/> elements to be included in the <see cref="T:System.Span`1" />.</param>
+        <summary>Creates a new <see cref="T:System.Span`1" /> object  from a specified number of <typeparamref name="T"/> elements starting at a specified memory address.</summary>
```

Comment:
The ReadOnlySpan xml needs to be fixed too:
[ReadOnlySpan Ctors](https://docs.microsoft.com/en-us/dotnet/api/system.readonlyspan-1.-ctor?view=netstandard-2.1#System_ReadOnlySpan_1__ctor_System_Void__System_Int32_)
https://github.com/dotnet/dotnet-api-docs/blob/3417fadcec76fa01f6b262ea7bcd999ae6b74a50/xml/System/ReadOnlySpan%601.xml#L107

It seems to have the correct length description, but uses the term, "object" rather than "elements". I prefer elements, but not sure which is best. cc @mairaw, @rpetrusha

Reply:
Good catch -- will do.

Reply:
Given the context, elements is better than objects, @ahsonkhan.